assets/js/modules/analytics/dashboard/dashboard-widget-analytics-adsense-top-pages.js

```diff
@@ -146,24 +144,39 @@ const isDataZero = () => {
  *
  */
 const getDataError = ( data ) => {
-	if ( ! data || ! data.error ) {
-		return false;
-	}
+	if ( data.code && data.message && data.data && data.data.status ) {
+		// Specifically looking for string "badRequest"
+		if ( data.data.reason && 'badRequest' === data.data.reason ) {
+			return AnalyticsAdSenseDashboardWidgetTopPagesTable.renderLayout(
+				<CTA
+					title={ __( 'Restricted metric(s)', 'google-site-kit' ) }
```

Comment:
We should use a different title for this message as this is more of an under-the-hood detail. Using the word "restricted" may also sound like the user has done something "forbidden" when they haven't.

Maybe something like "Connect Analytics with AdSense for more data" to call the user to action?